### PR TITLE
Clear old slot data when loading calendar page

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
Previous attempt at fixing this didn't actually clear the old slot data, just reset the months.

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Slots are correct

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/api/index.js [hunk 1]
```diff
@@ -293,7 +293,7 @@ export function getRequestLimits(facilityId, typeOfCareId) {
 export function getAvailableClinics(facilityId, typeOfCareId, systemId) {
   let promise;
   if (USE_MOCK_DATA) {
-    if (facilityId === '983A6') {
+    if (facilityId === '983') {
```
Comment (narin) [inline src/applications/vaos/api/index.js:296]: Is this related to slots? Or just a separate fix you caught?

**Review by narin — APPROVED** [review]

File: src/applications/vaos/api/index.js (hunk 1 above)
Comment (jbalboni) [reply src/applications/vaos/api/index.js:296]: The DS flow wasn't working for me locally, this fixed it.
